python/cudf/cudf/io/__init__.py

```diff
@@ -13,3 +14,5 @@
     read_parquet_metadata,
     write_to_dataset,
 )
+
+HostBuffer = cudf._lib.io.utils.HostBuffer
```

Comment:
I could go ```from cudf._lib.io.utils import HostBuffer  # noqa``` , but think we'd need the noqa tag otherwise flake8 will complain about unused imports. (I was trying to be consistent here  with https://github.com/rapidsai/cudf/pull/4870#discussion_r406512712 =)